src/Parsers/ASTAlterQuery.h

```diff
@@ -247,6 +247,7 @@ class ASTAlterQuery : public ASTQueryWithTableAndOutput, public ASTQueryWithOnCl
     }
 
     const char * getQueryKindString() const override { return "Alter"; }
```

Comment:
Now we can use `magic_enum` (intorduced https://github.com/ClickHouse/ClickHouse/pull/28659) to implement `getQueryKindString`? Or just delete this method and use `getQueryKind` for existing calls

Reply:
The only remaining place with the method is in https://github.com/ClickHouse/ClickHouse/blob/3bb9da028634d9d9f9396ac281a666fad09aa64f/src/Interpreters/executeQuery.cpp#L274

It looks like it may be easily replaced by a new method if you'd suggest how.

Reply:
Ok, let's try like this [`ae7e569` (#33294)](https://github.com/ClickHouse/ClickHouse/pull/33294/commits/ae7e5691b83ac05444e722dbfda97965673e6081). I've pushed this commit into this PR to check tests tests are ok